src/renderer/caching_shaper.rs

```diff
@@ -67,16 +67,16 @@ impl ExtendedFontFamily {
 
         None
     }
-}
-
-fn to_family(x: ExtendedFontFamily) -> FontFamily {
-    let mut y = FontFamily::new();
 
-    for font in x.fonts {
-        y.add_font(font);
+    pub fn to_family(self) -> FontFamily {
```

Comment:
Maybe take self as a reference?

Reply:
probably a mutable reference

Reply:
> Maybe take self as a reference?

@Kethku so self here takes ownership, but it doesn't have implications so does it really differ from this in this case?

```rust
pub fn to_family(&self) -> FontFamily {
    let mut new_family = FontFamily::new();

    for font in &self.fonts {
        new_family.add_font(font.to_owned());
    }

    new_family
}
```

> probably a mutable reference

any particular reason?

Reply:
It was a dumb suggestion. I was misunderstanding the purpose of the function. Ignore